src/toil/batchSystems/gridengine.py

```diff
@@ -90,6 +91,8 @@ def getjobexitcode(sgeJobID):
 
         process = subprocess.Popen(args, stdout = subprocess.PIPE,stderr = subprocess.STDOUT)
         for line in process.stdout:
+            logger.debug(args)
```

Comment:
I would agree that they are not needed. I thought I deleted these commits when I rebased interactively but when I squashed I must have incorporated them.  Is the appropriate step to submit an additional commit that removes those lines?